homeassistant/components/binary_sensor/wink.py

```diff
@@ -61,7 +61,7 @@ def _pubnub_update(self, message, channel):
                 json_data = message
             self.wink.pubnub_update(json.loads(json_data))
             self.update_ha_state()
-        except (AttributeError, KeyError):
+        except (AttributeError, KeyError, AttributeError):
```

Comment:
@MartinHjelmare wow you're right I was "fixing" it so quick I didn't even notice it was already there. It is really strange though because that is were the error was coming from. Full log below. Shouldn't an exception in a dependency get raised to HA?

```bash
Nov 06 15:34:36 hass hass[10732]: File "/home/pi/.virtualenvs/hass_dev/lib/python3.4/site-packages/homeassistant-0.33.0.dev0-py3.4.egg/homeassistant/components/wink.py", line 113, in _pubnub_update
Nov 06 15:34:36 hass hass[10732]: self.wink.pubnub_update(message)
Nov 06 15:34:36 hass hass[10732]: File "/home/pi/.virtualenvs/hass_dev/config/deps/pywink/devices/standard/__init__.py", line 154, in pubnub_update
Nov 06 15:34:36 hass hass[10732]: self._update_state_from_response(json_response)
Nov 06 15:34:36 hass hass[10732]: File "/home/pi/.virtualenvs/hass_dev/config/deps/pywink/devices/standard/__init__.py", line 149, in _update_state_from_response
Nov 06 15:34:36 hass hass[10732]: if outlet.get('outlet_id') == str(self.device_id()):
Nov 06 15:34:36 hass hass[10732]: AttributeError: 'str' object has no attribute 'get'
```